apps/rosen/app/(bridge)/ChooseWalletModal.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  Grid,
+  Tooltip,
+} from '@rosen-bridge/ui-kit';
+import { Wallet, WalletInfo } from '@rosen-ui/wallet-api';
+
+interface ChooseWalletModalProps {
+  open: boolean;
+  handleClose: () => void;
+  setSelectedWallet: ((wallet: Wallet) => Promise<void>) | undefined;
+  chainName: string;
+  supportedWallets: WalletInfo[];
+  availableWallets: Wallet[];
+}
+
+/**
+ * modal to allow user to choose a wallet
+ *
+ * @param open
+ * @param handleClose
+ * @param chainName
+ * @param supportedWallets
+ */
+export const ChooseWalletModal = ({
+  open,
+  handleClose,
+  setSelectedWallet,
+  chainName,
+  supportedWallets,
+  availableWallets,
+}: ChooseWalletModalProps) => {
+  const handleConnect = async (wallet: Wallet) => {
+    setSelectedWallet && (await setSelectedWallet(wallet));
+    handleClose();
+  };
+
+  return (
+    <Dialog onClose={handleClose} open={open} maxWidth="tablet">
+      <DialogContent>
+        <DialogContentText>
+          Please choose any of the supported wallets for {chainName} chain.
+        </DialogContentText>
+        <Grid
+          container
+          justifyContent="center"
+          sx={{ pt: (theme) => theme.spacing(1) }}
+        >
+          {supportedWallets.map(
+            ({ icon: WalletIcon, label, link, name }, index) => (
+              <Tooltip title={name} key={label}>
+                <Grid
+                  item
+                  sx={{
+                    width: '8rem',
+                    height: '8rem',
+                    border: (theme) => `1px solid ${theme.palette.divider}`,
+                    borderRadius: 1,
+                    padding: (theme) => theme.spacing(1),
+                    margin: (theme) => theme.spacing(1),
+                    cursor: 'pointer',
+                    flexDirection: 'column',
```

Comment:
Use `Grid` component props instead of these CSS props. Please check if the same issue exists in your other changes.